packages/devtools_app/lib/src/shared/table.dart

```diff
@@ -815,14 +818,16 @@ class _Table<T> extends StatefulWidget {
   int get numSpacers => max(0, columns.length - 1);
 
   @override
-  _TableState<T> createState() => _TableState<T>();
+  _TableState<T> createState() => _TableState<T>(rowItemExtent: rowItemExtent);
 }
 
 class _TableState<T> extends State<_Table<T>> with AutoDisposeMixin {
+  _TableState({this.rowItemExtent});
```

Comment:
never pass parameters into state objects. You can access the class widget class member from the state object via `widget.rowItemExtent`

Reply:
this is a good life lesson thank you :)

Reply:
I'm surprised the no-logic in createState lint didn't warn about this.